glyphs-reader/src/glyphdata.rs

```diff
@@ -73,6 +73,133 @@ pub enum Subcategory {
     Other,
 }
 
+/// The script of a given glyph
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[repr(u8)]
+pub enum Script {
```

Comment:
This is lovely for being almost but not quite entirely a subset of the scripts defined in ICU 😒

Fun additions here include "Alchemical" and "Dentistry"?


I presume referring to... https://en.wikipedia.org/wiki/Alchemical_Symbols_(Unicode_block) and https://en.wikipedia.org/wiki/Palmer_notation?

Reply:
TIL lol